apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/common/ClaimDao.java

```diff
@@ -123,28 +120,8 @@ public Object getEntityById(ResourceTypeV2<?> type, String id) {
 
   @VisibleForTesting
   Predicate createDateRangePredicate(
```

Comment:
this is now just a one line method, are you keeping it so its easy to mock in tests?

Reply:
exactly